jsongraph.js

```diff
@@ -49,8 +80,10 @@ JSONGraph.prototype.forEachStatement = function (callback)
 	for(let i = 0; i < this.triples.length; i++)
 	{
 		let t = this.triples[i];
-		
-		callback(t[0], t[1], t[2], t[3]);
+		let object = t[2];
+		object = object && object.includes(`"`) ? this.parseLiteral(object) : object;
```

Comment:
Perhaps replace with a direct char test?

Reply:
I'm not sure if I follow, but I believe the `.includes` could be replaced with a `.startsWith`. Does that address your concern @srfiorini ?